# HG changeset patch
# User Ken Brown <kbrown@cornell.edu>
# Date 1426358732 14400
#      Sat Mar 14 14:45:32 2015 -0400
# Node ID abc756d38a96bd7959869ca47d66590df7c19baf
# Parent  31b8bc6744c439be656091ee45d3a27dd710798d
Use Unix line endings on Cygwin

diff -r 31b8bc6744c4 -r abc756d38a96 src/encoding.d
--- a/src/encoding.d	Sat Mar 14 10:29:23 2015 -0400
+++ b/src/encoding.d	Sat Mar 14 14:45:32 2015 -0400
@@ -2573,7 +2573,7 @@
   unused name; unused context;
   pushSTACK(unbound);           /* :charset */
  #endif /* ENABLE_UNICODE */
- #if defined(WIN32) || (defined(UNIX) && (O_BINARY != 0))
+ #if defined(WIN32) || (defined(UNIX) && (O_BINARY != 0) && !defined(UNIX_CYGWIN32))
   pushSTACK(S(Kdos));           /* :line-terminator */
  #else
   pushSTACK(S(Kunix));          /* :line-terminator */
